packages/ckeditor5-list/src/listproperties/listpropertiesui.ts

```diff
@@ -299,9 +300,12 @@ function getStyleButtonCreator( {
 
 		button.set( { label, icon, tooltip } );
 
-		listStyleCommand.on( 'change:value', () => {
-			button.isOn = listStyleCommand.value === type;
-		} );
+		const refreshIsOn = () => {
+			button.isOn = getAllListStyleAliases( listStyleCommand.value ).includes( type );
+		};
+
+		refreshIsOn();
+		listStyleCommand.on( 'change:value', refreshIsOn );
```

Comment:
Maybe would be good to bind it using editors API?

Something like this:

```
button.bind( 'isOn' ).to( listStyleCommand, 'value', value => getAllListStyleAliases( value! ).includes( type ) );
```

WDYT?

Reply:
hm... good idea, I'll check that

Reply:
works as expected, thx. Fixed https://github.com/ckeditor/ckeditor5/pull/17627/commits/94d254372c81a57aba63606b78a35cbe4fdc5aee